api/command.go

```diff
@@ -86,6 +86,32 @@ func listCommands(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(model.CommandListToJson(commands)))
 }
 
+func getCommands(c *Context) []*model.Command {
+	commands := make([]*model.Command, 0, 32)
+	seen := make(map[string]bool)
```

Comment:
Why is seen only for built in commands?  would make sense for it to be the other way around.